tests/components/controlBar-test.js

```diff
@@ -1624,4 +1650,79 @@ describe('ControlBar', function () {
     var logo = TestUtils.scryRenderedDOMComponentsWithClass(DOM, 'oo-logo');
     expect(logo.length).toBe(0);
   });
+
+  describe('Tab Navigation', function() {
+    var eventMap, ctrlBarElement, focusableElements, mockEvent;
+
+    beforeEach(function() {
+      baseMockProps.skinConfig.buttons.desktopContent = [
+        { "name": "playPause", "location": "controlBar", "whenDoesNotFit": "keep", "minWidth": 45 },
+        { "name": "volume", "location": "controlBar", "whenDoesNotFit": "keep", "minWidth": 240 },
+        { "name": "fullscreen", "location": "controlBar", "whenDoesNotFit": "keep", "minWidth": 45 },
+      ];
+      // Mock addEventListener on document object since TestUtils.Simulate will not work in this case
+      eventMap = {};
+      document.addEventListener = jest.genMockFn().mockImpl(function(event, cb) {
```

Comment:
Do we need to restore this function in afterEach?

Reply:
Will check, we probably do